mobile/src/main/java/com/boswelja/devicemanager/watchmanager/WatchManager.kt

```diff
@@ -109,19 +108,16 @@ class WatchManager internal constructor(
     suspend fun forgetWatch(context: Context, watch: Watch) {
         forgetWatch(
             WatchBatteryStatsDatabase.getInstance(context),
-            WidgetDatabase.getInstance(context),
             watch
         )
     }
 
     internal suspend fun forgetWatch(
         batteryStatsDatabase: WatchBatteryStatsDatabase,
-        widgetDatabase: WidgetDatabase,
         watch: Watch
     ) {
         withContext(Dispatchers.IO) {
             batteryStatsDatabase.batteryStatsDao().deleteStatsForWatch(watch.id)
-            widgetDatabase.widgetDao().removeWidgetsFor(watch.id)
```

Comment:
There's a regression in these functions, where removing a watch won't clear settings for it's associated widgets.